sourcegraph/handbook
# Limit content on the org page

Resolves #818 by limiting the amount of content that gets pulled in on the org page. Builds on #895 by extending the idea of autogenerated content from `team.yml` to other teams.

### To Test
- Check out the branch, and head over to `team/org_chart`. Ensure that only the team members are listed and no other page information is displayed.
- Check that the updated directions and links to edit files are up-to-date and make sense.
